2-mandatory/2-financial-times.js

```diff
@@ -24,6 +44,13 @@ function titleWithFewestWords(allArticleTitles) {
 */
```

Comment:
The next few exercises look good as well!
I would recommend you practice all of these using array methods now 😄